e2e/custom-plugins/grafana-extensionstest-app/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "./.config/tsconfig.json"
```

Comment:
As mentioned above I think we can extend `@grafana/plugin-configs/tsconfig.json` rather than introduce the config files in this app.